avidemux_fr.ts

```diff
@@ -3615,7 +3620,7 @@ Incompatibilité d&apos;installation</translation>
     <message>
         <location line="+1"/>
         <source>_Saturation:</source>
-        <translation type="unfinished"></translation>
+        <translation>-Saturation</translation>
```

Comment:
Could you please restore the accelerator (the underscore), destroyed by an apparent typo?

Reply:
the correction is done ([5ca74d7](https://github.com/mean00/avidemux2_i18n/pull/20/commits/5ca74d7bcb2161f6674e06afbe8d89dd4cd3373e)